### PR TITLE
adds checks for icons and toc attributes

### PR DESCRIPTION
**Pull request by Levi-Leah**

### CODE REVIEW

**Comment by Levi-Leah** [comment]
@adahms 
this should throw errors for icons and toc attributes

**Review by adahms — APPROVED** [review]
Thanks @Levi-Leah - I can confirm that the icons and toc tests are working as expected.

One question - in case someone includes these in their pantheon2.yml file, should we check that as well for any references?

If you'd like to go ahead and merge this, we can address that question in a separate issue.

**Comment by Levi-Leah** [comment]
@adahms 
Can you clarify about the inclusion of these attributes in the pv2.yml? Do you mean if files that contain these attributes are part of pv2.yml?

**Comment by adahms** [comment]
@Levi-Leah - Gah, sorry - not the pantheon2.yml file, but the attributes file. ><

**Comment by Levi-Leah** [comment]
@adahms
right now I don’t parse attributes file for anything related to the checks but it should be easy to set up. That being said, if none of the modules/assemblies actually call for these attributes, would flagging it in the attributes file be necessary? Would having in attributes file break smth?

**Comment by adahms** [comment]
@Levi-Leah - very true. Perhaps the only thing I'm worried about are these attributes that change the structure of the document. Anything a user specifies in the pantheon2.yml file gets injected into every assembly and module, meaning that if a user specifies the 'toc' attribute there, everything gets it.

**Comment by Levi-Leah** [comment]
@adahms 
I added toc/icons validation for the attributes file. please take a look when you get the time

**Review by adahms — CHANGES_REQUESTED** [review]
Hey @Levi-Leah - PCMD is now detecting cases of these attributes in the pantheon2.yml file. Added a comment about some double reporting in results, otherwise, everything's working as expected.

File: PantheonCMD/pcvalidator.py [hunk 1]
```diff
@@ -64,5 +68,7 @@ def validation(files_found, modules_found, assemblies_found):
             stripped = Regex.INTERNAL_IFDEF.sub('', stripped)
             nesting_in_assemblies_check(report, stripped, path)
             add_res_section_assembly_check(report, stripped, path)
+            icons_check(report, stripped, path)
```
[the remark this answers is not in the document]
Comment (Levi-Leah) [reply PantheonCMD/pcvalidator.py]: fixed

**Pull request merged**